### ISSUE TITLE
Run example error

### ISSUE THREAD

**Issue by 1633232731** [ChristoffelDoorman/network-rewiring-rl#2]
Run command 
```shell
docker exec -it relnet-manager /bin/bash -c "source activate ucfadar-relnet && python relnet/experiment_launchers/run_rnet_dqn.py"
```

Response an error:
```python
Traceback (most recent call last):
  File "relnet/experiment_launchers/run_rnet_dqn.py", line 6, in <module>
    from relnet.agent.rnet_dqn.rnet_dqn_agent import RNetDQNAgent
  File "/relnet/relnet/agent/rnet_dqn/rnet_dqn_agent.py", line 11, in <module>
    from relnet.agent.rnet_dqn.q_net import NStepQNet, greedy_actions
  File "/relnet/relnet/agent/rnet_dqn/q_net.py", line 6, in <module>
    from relnet.agent.fnapprox.gnn_regressor import GNNRegressor
ModuleNotFoundError: No module named 'relnet.agent.fnapprox'
```

**Comment by ChristoffelDoorman** [comment]
This is now fixed in [f45aad8](https://github.com/ChristoffelDoorman/network-rewiring-rl/commit/f45aad82f3cf583f62eed6224ef17d7cdc59f911).

**Comment by 1633232731** [comment]
Thank you！I will try later.
